src/game/game.gd

```diff
@@ -20,3 +20,11 @@ func addCharacter(networkId: int) -> void:
 	characterPosition.y = rng.randi_range(100, 500)
 	add_child_below_node(characterNode, newCharacter) ## Add node to scene
 	newCharacterResource.setPosition(characterPosition) ## Apply position
+
+func removeCharacter(networkId: int) -> void:
+	#print_debug("game: removing character", networkId)
+	var characterNode = Characters.getCharacterNode(networkId)
+	characterNode.queue_free()
```

Comment:
@nicemicro queue_free() is being called here, should I put it in `removeCharacterNode` instead?

Reply:
ohh, I missed this here.
It is an interesting question though. My first thought was that it should be queue_free()d when the reference is removed from the dictionary, but on a second thought, it is the game scene which is responsible for handling stuff on the map... so both makes sense.